internal/pkg/partition/format.go

```diff
@@ -57,9 +57,7 @@ func zeroPartition(devname string) (err error) {
 
 	defer part.Close() //nolint:errcheck
 
-	_, err = part.Wipe()
-
-	return err
+	return part.FastWipe()
```

Comment:
Should we just use it if Size == 0?

Reply:
I mean how it was before we introduced the changes, or is Size now gone?

Reply:
Size got moved to partition options, and we've only format options here now, I'm not sure if this is the same path used when wiping a disk with `wipe: true`